class.phpmailer.php

```diff
@@ -1474,8 +1496,8 @@ protected function smtpSend($header, $body)
         }
 
         // Attempt to send to all recipients
-        foreach (array($this->to, $this->cc, $this->bcc) as $togroup) {
-            foreach ($togroup as $to) {
+        if (!empty($this->envelope_to)) {
+            foreach ($this->envelope_to as $to) {
```

Comment:
If I read this correctly: as soon as I specify at least 1 "envelope_to" recipient, then the mail is delivered only these "envelope_to" recipients, and none of the "to", "cc", and "bcc" recipients? This should be made clear in the code comments here, and elsewhere where "envelope_to" recipients appear.

In the same PR, a new or modified example in the "examples" directory would be welcome too.